crates/ironrdp/Cargo.toml

```diff
@@ -47,18 +47,17 @@ ironrdp-dvc = { workspace = true, optional = true }
 ironrdp-rdpdr = { workspace = true, optional = true }
 ironrdp-rdpsnd = { workspace = true, optional = true }
 ironrdp-displaycontrol = { workspace = true, optional = true }
+x509-cert = "0.2.5"
```

Comment:
question: Do we really need this new dependency in the `ironrdp` crate? This used to be a dev dependency for the example, but now we have the helper from `ironrdp-server`.

Reply:
that's a mistake! thanks